packages/react-native-reanimated/src/layoutReanimation/web/animationsManager.ts

```diff
@@ -50,6 +50,35 @@ function checkUndefinedAnimationFail(
   return true;
 }
 
+function maybeReportOverwrittenProperties(
+  keyframe: string,
+  styles: CSSStyleDeclaration
+) {
+  const propertyRegex = /([a-zA-Z-]+)(?=:)/g;
+  const animationProperties = new Set();
+
+  let match;
+  while ((match = propertyRegex.exec(keyframe)) !== null) {
+    animationProperties.add(match[1]);
+  }
```

Comment:
You can simply use matchAll instead of looping over all match:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/matchAll

```
const matches = str.matchAll(regexp);
```

Reply:
Done in [c144817](https://github.com/software-mansion/react-native-reanimated/pull/6064/commits/c144817fd72be2064d2df9d5c66cd5663776cc32).